pkg/operator/bootstrapteardown/bootstrap_teardown_controller.go

```diff
@@ -188,6 +203,14 @@ func (c *BootstrapTeardownController) canRemoveEtcdBootstrap() (bool, bool, erro
 	}
 }
 
+func (c *BootstrapTeardownController) getExpectedEtcdSize() (int, error) {
+	nodes, err := c.nodeLister.List(labels.Set{"node-role.kubernetes.io/master": ""}.AsSelector())
```

Comment:
this is definitive? or does it change? I thought that expected was going to be based on master replicas. So if we have a cluster that will eventually have 3 nodes. Will we see 1 node as expected until the other nodes rollout? Expected should be explicit. install-config master replicas was that data point we talked about wasn't it?

Reply:
With a three-node cluster we never want the single master gate open.

Reply:
I will re-work